sosw/components/test/unit/test_dynamo_db.py

```diff
@@ -318,5 +319,12 @@ def test_patch__transfers_attrs_to_remove(self):
                                                           condition_expression='attribute_exists hash_col')
 
 
+    def test_sleep_db(self):
```

Comment:
You could try to write better tests. For example, one test when it sleeps and one when it doesn't